build/dsc/resource.rb

```diff
@@ -110,7 +110,12 @@ def ensure_value
           'present'
         elsif ensure_property.values.any? { |v| v.casecmp('enable') == 0 }
           'enable'
+        elsif ensure_property.values.any? { |v| v.casecmp('published') == 0 }
```

Comment:
Does any of the above change if `Published` and `Draft` are treated as `Present` inside the DSC Resource code? Looking through the [code](https://github.com/PowerShell/xSCSMA/blob/dev/DSCResources/MSFT_xRunbookDirectory/MSFT_xRunbookDirectory.psm1#L155-L200), it's either if `Absent` remove **OR** ensure it's `Present` and set the state of the Runbook as `Published` or `Draft`. In essence they both mean its `Present`.

My instinct is that this is abuse of the Ensure parameter by the DSC Resource and that this should be a Ensure and a State parameter, but they moved from that implementation to this so it's unlikely they would change back.